interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/FragmentParameterValidator.scala

```diff
@@ -35,27 +38,66 @@ class FragmentParameterValidator(
       fragmentParameter.name
     )
 
+    val validationExpressionResult =
+      compileExpressionValidator(fragmentParameter, validationContext(fragmentParameter.name))
+
     List(
       unsupportedFixedValuesValidationResult,
       fixedValuesListValidationResult,
-      fixedExpressionsValidationResult
-    ).sequence.map(_ => ())
+      fixedExpressionsValidationResult,
+      validationExpressionResult
+    ).sequence.map(_.flatten)
+  }
+
+  private def compileExpressionValidator(
+      fragmentParameter: FragmentParameter,
+      typ: TypingResult
+  )(implicit nodeId: NodeId) = fragmentParameter.valueCompileTimeValidation match {
```

Comment:
Let's use just `fragmentParameter.valueCompileTimeValidation.flatMap(...).sequence` instead of pattern matching

Reply:
I can't find my comment about the type, but let's use `Option` instead of `List`  - I see only one optional validator returned by this method